packages/td-tools/test/AssetInterfaceDescriptionTest.ts

```diff
@@ -125,7 +153,7 @@ class AssetInterfaceDescriptionUtilTest {
             .to.have.lengthOf(1);
 
         // filter Modbus and HTTP and submodel only
-        const td3 = this.assetInterfaceDescriptionUtil.transformToTD(modelAID, `{"title": "myTitle"}`, "Modbus|HTTP");
+        const td3 = this.assetInterfaceDescriptionUtil.transformAAS2TD(modelAID, `{"title": "myTitle"}`, "Modbus|HTTP");
```

Comment:
is it also possible to filter by interface name?

Reply:
I think that's what it is doing...
We currently have for example the following term in JSON

`"idShort": "Interface1 (Modbus; s. slides from 21.6.2022)"`

The regex "Modbus|HTTP" look into _this_ `idShort` and tries to find either Modbus or HTTP...

One could also use "Interface1" for the regex to select Modbus.

Reply:
ok, perfect. From my view we can merge this PR